src/annotation.js

```diff
@@ -92,7 +92,7 @@ export default {
     }
   },
 
-  destroy(chart) {
+  afterDestroy(chart) {
```

Comment:
As we have not made a major version jump yet and thus should be compatible with Chart.js v3, maybe we should just keep both for now=

Reply:
Aah yes totally forgot about that 😓

Reply:
@kurkle @LeeLenaleee not needed because the plugin has the dependency with chartjs 3.7.0 and afterDestroy has been introduced in version 3.7.0